packages/cli/lib/project.js

```diff
@@ -639,31 +645,20 @@ class Project extends EventEmitter {
         let ebook = await new Epub(bookData, configData);
         let data = await ebook.generate();
         let epub = await new Build(data, spinner);
-        let dir = "html/";
-        let epubfile = file !== undefined ? file : "static/downloads/output";
+        let dir = "html";
         let dirCreated = fs.existsSync(dir);
         if (dirCreated) {
-          let fileName = path.basename(epubfile);
-          if (fileName.indexOf(".") !== -1) {
-            file = fileName.split(".");
-            fileName = file[0];
-          }
-          let filePath = epubfile.substring(0, epubfile.lastIndexOf("/"));
-          filePath =
-            filePath !== undefined && filePath !== ""
-              ? filePath
-              : "static/downloads";
-          fileName =
-            fileName !== undefined && fileName !== "" ? fileName : "output";
+          const fileNamePath = file !== undefined ? file : path.join("static", "downloads", "output");
+          const filePath = path.dirname(fileNamePath);
+          const fileName = path.basename(fileNamePath);
           await fs.ensureDir(filePath);
           let cover =
             epub.data.cover !== undefined
               ? epub.data.cover.replace("http://localhost:1313/img/", "")
               : "";
-          cover = cover !== "" ? `--epub-cover-image=static/img/${cover}` : "";
-          const fileNamePath = path.join(filePath, fileName);
-          let args = `-f html-native_divs+native_spans -t epub html/epub.xhtml -o ${fileNamePath}-mobi.epub --epub-metadata=html/dc.xml ${cover} --template=html/template.xhtml --css=${outputDir}/css/epub.css -s`;
-          execSync(`pandoc ${args}`);
+          cover = cover !== "" ? `${path.join("static", "img", cover)}` : "";
```

Comment:
Do you think this expression is a bit odd, it reads "if the cover image **is not an _empty string_**, use the path to the cover otherwise **use an _empty string_**."?